include/api/CDataFrameAnalysisSpecification.h

```diff
@@ -156,9 +162,13 @@ class API_EXPORT CDataFrameAnalysisSpecification {
     //!   2. disk is used (only one partition needs to be loaded to main memory)
     void estimateMemoryUsage(CMemoryUsageEstimationResultJsonWriter& writer) const;
 
+    TOStreamSPtr persistStreamSupplier() const;
```

Comment:
Seems like this is better named as:
```suggestion
    TOStreamSPtr persistStream() const;
```

Reply:
Done.